conda/meta.yaml

```
@@ -1,28 +1,33 @@
 package:
   name: scwidgets
-  version: { { GIT_DESCRIBE_TAG } }
+  version: "1"
```

Is there a reason for not using the Git tag here? (note that the previous syntax is wrong, there should be no space in the double parenthesis).

```suggestion
  version: {{ GIT_DESCRIBE_TAG }}
```

---

There is not. I keep changing it to test locally and then accidentally committing thanks for pointing out.